src/components/Auction.jsx

```diff
@@ -1,11 +1,15 @@
-import React from "react";
+import { useContext } from "react";
 import { Routes } from "../types/RouteTypes";
+import Detailmodal from "./modals/Detailmodal";
+import BaseModal from "./modals/baseModal";
 import { useNavigate } from "react-router-dom";
 import Timer from "./Timer.jsx";
 import { RiCoinLine } from "react-icons/ri";
+import { AppContext } from "../context/AppContext";
 
 const Auction = ({ id, name, expiration, highest_bid, image }) => {
-    const navigate = useNavigate();
+    const { setModal } = useContext(AppContext);
+
     return (
         <>
```

Comment:
Aangezien <article> hoofdelement is, worden geen "siblings" gerendered en mag de fragment (<></>) weggelaten worden